### ISSUE TITLE
Unable to add Item quantity

### ISSUE THREAD

**Issue by lahkun** [opensourcepos/opensourcepos#2928]
Hi,

In the sales panel, i am not able to edit/add item quantity. It shows the quantity label but not showing edit box. Can anyone suggest configuration?
I use version 3.3.1. Thanks

**Comment by odiea** [comment]
Yes. You cannot edit an Item quantity that has a serial number. Serial numbers should be unique.
